core/src/main/java/org/frankframework/parameters/Parameter.java

```diff
@@ -563,10 +563,14 @@ public Object getValue(ParameterValueList alreadyResolvedParameters, Message mes
 		}
 
 		if (result instanceof Message) { //we just need to check if the message is null or not!
-			if (Message.isNull((Message)result)) {
+			if (Message.isNull((Message) result)) {
```

Comment:
Shall we just assign `Message message = (Message) result;` here? So we don't have to cast all the time?

Reply:
Ey yes, sure, it happened because 2 commits on the same code. I'll fix that.